server_admin/topics/authentication/webauthn.adoc

```diff
@@ -17,38 +17,33 @@ The setup procedure of WebAuthn support for 2FA is the following :
 
 . Click *Authentication* in the menu.
 . Click the *Required Actions* tab.
-. Click *Register*.
-. Click the *Required Action* drop-down list.
-. Click *Webauthn Register*.
-. Click *Ok*.
+. Toggle the *Webauthn Register* switch to *ON*.
 
-Mark the *Default Action* checkbox if you want all new users to be required to register their WebAuthn credentials.
+Toggle the *Default Action* switch to *ON* if you want all new users to be required to register their WebAuthn credentials.
 
 [[_webauthn-authenticator-setup]]
 ===== Adding WebAuthn authentication to a browser flow
 
 . Click *Authentication* in the menu.
 . Click the *Browser* flow.
-. Click *Copy* to make a copy of the built-in *Browser* flow.
+. Select *Duplicate* from the "ellipses button" to make a copy of the built-in *Browser* flow.
 . Enter a name for the copy.
-. Click *Ok*.
-. Click the _Actions_ link for *WebAuthn Browser Browser - Conditional OTP* and click _Delete_.
-. Click *Delete*.
+. Click *Duplicate*.
+. Click on the *🗑️* (trash can icon) of the "WebAuthn Browser Browser - Conditional OTP" and click *Delete*.
```

Comment:
I don't see "WebAuthn Browser Browser - Conditional OTP" even if I do it the way you describe with the ellipses.
Can you verify if something is missing.